components/Vision.js

```diff
@@ -4,7 +4,10 @@ import Frame from "./Frame";
 
 const Vision = () => {
 	return (
-		<Row className='font-lexend flex text-acm-black justify-between items-center w-11/12 mx-auto pt-4'>
+		<Row
+			className='font-lexend flex text-acm-black justify-between items-center w-11/12 mx-auto pt-4'
+			style={{ marginTop: "12vh" }}
```

Comment:
try this using tailwind, it is possible you can look thru their documentation or thru other code in this repo

Reply:
yessir https://stackoverflow.com/a/69573881